administrator/components/com_patchtester/src/Model/PullModel.php

```diff
@@ -22,7 +22,7 @@
 use Joomla\Component\Patchtester\Administrator\GitHub\GitHub;
 use Joomla\Component\Patchtester\Administrator\GithubCredentialsTrait;
 use Joomla\Component\Patchtester\Administrator\Helper\Helper;
-use Joomla\Filesystem\File;
+use Joomla\CMS\Filesystem
```

Comment:
@alikon could you also patch the next line `use Joomla\CMS\Filesystem\Folder;`

Reply:
done 
i was unable to replicate on my local